### ISSUE TITLE
Upgrade to terraform-plugin-sdk v2.25.0

### ISSUE THREAD

**Issue by jbergknoff** [mongodb/terraform-provider-mongodbatlas#1080]
Hi, because of a change between Terraform 1.3.7 and 1.3.8, if you import a `mongodbatlas_cluster` and then run a plan, you will always see a difference:

```
~ resource "mongodbatlas_cluster" "foo" {
    id = ...
    name = ...

  - timeouts {}
}
```

I was also seeing this with `aws_security_group`, so reported it as a bug in Terraform (https://github.com/hashicorp/terraform/issues/32661). However, the response was that Terraform's behavior is more picky, but more correct, now. A fix for this landed in `terraform-plugin-sdk` [v2.25.0](https://github.com/hashicorp/terraform-plugin-sdk/releases/tag/v2.25.0). Please upgrade when you have a chance. Thanks!

**Comment by Zuhairahmed** [comment]
hi @jbergknoff thanks for flagging this issue. just confirming that we are currently on terraform-plugin-sdk v2.24.1. after we upgrade to v2.25.0 will let you know.

**Comment by Zuhairahmed** [comment]
@jbergknoff just FYI we are working on this upgrade now, will be include as part of v1.8.2 release to be published to Terraform Public Registry in ~2 weeks. https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1100

**Comment by Zuhairahmed** [comment]
hi @jbergknoff we have released v1.8.2 now which includes this update. closing this issue, but feel free to re-open if you need anything else here: https://registry.terraform.io/providers/mongodb/mongodbatlas/latest
